packages/renderer-protocol/inverseRpc/services/friendRequestService.ts

```diff
@@ -73,7 +79,32 @@ export function registerFriendRequestKernelService(port: RpcServerPort<RendererP
     },
 
     async rejectFriendRequest(req, _) {
-      return {}
+      try {
+        // Handle reject friend request
+        const rejectFriend = await rejectFriendRequest(req)
+
+        let rejectFriendRequestReply: RejectFriendRequestReply = {}
+
+        // Check response type
+        if (rejectFriend.reply?.friendRequest) {
+          rejectFriendRequestReply = {
+            message: {
+              $case: 'reply',
+              reply: rejectFriend.reply
+            }
+          }
+        } else {
+          rejectFriendRequestReply = buildResponse(rejectFriend)
+        }
```

Comment:
Shouldn't this be simply?
```suggestion
        const rejectFriendRequestReply: RejectFriendRequestReply = buildResponse(rejectFriend)
```

Reply:
🥲